### PR TITLE
October blog meeting minutes

### PR DESCRIPTION
**Pull request by surfersol**

### CODE REVIEW

**Comment by bitkarrot** [comment]
@saucy-tech please to review tags. 

@bliotti  Good idea to add the meeting minutes to the website, I don't know why we didn't do it before, as there is a repo here..... https://github.com/plebnet-dev/meeting-notes

Either we consolidate everything into the website or add a copy for October Meeting to the meeting-notes repo as well for easy reference, as the entire history is currently over there. Opinions on organizing this welcome, especially on tags, @saucy-tech, @surfersol @bliotti

File: src/content/post/surfersol_blogpost.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  - lightning-network
+  - Git workflow
+  - Blogpost
+---
+
+# October's Meeting Minutes
+
+Meeting’s announcements and discussions
+- [Nostr](https://testnet.plebnet.dev/nostrrelay/1) 
+| [Resources](https://metadata.plebnet.dev/)
```
Comment (bitkarrot) [inline src/content/post/surfersol_blogpost.md:18]: Update [Resources] to [Backup your Nostr Metadata]

File: src/content/post/surfersol_blogpost.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+publishDate: 2023-11-01T00:00:00Z
+title: October's Meeting Minutes
+image: /images/NovemberNOSTR.webp
+description: October's Meeting Minutes
+category: Beginner
+author: SurferSol 
+tags:
+  - lightning-network
```
Comment (bitkarrot) [inline src/content/post/surfersol_blogpost.md:9]: @saucy-tech please comment on tags

File: src/content/post/surfersol_blogpost.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+- Future Projects – GetAlby to announce a long term project with collaboration with Plebnet.dev
+
+- Recognizing our Active Contributors - Blogging Bitcoin, Saucy_tech, Tonyboi, Coinguybri, Fenix, Ash / Smash, BitKarrot
+
+- [Nostr use cases and benefits and Nostr matrix client list](https://nostorg.github.io/clients/)
+
+- Contributor’s projects outside of Plebnet.dev of personal development progress
+
+- [Completion of a LNBITS or GetAlby Pull Request would gets a free 2024 Bitcoin Tix Open Source](https://b.tc/conference/opensource)
```
Comment (bitkarrot) [inline src/content/post/surfersol_blogpost.md:30]: Not sure if getAlby is on the list. I think its only LNBits but please cross check. thanks

**Review by bitkarrot — CHANGES_REQUESTED** [review]
review made, please see requested changes @surfersol . Thanks!

File: src/content/post/surfersol_blogpost.md (hunk 2 above)
Comment (saucy-tech) [reply src/content/post/surfersol_blogpost.md:9]: I'd recommend to use 
```
category: News and Updates
tags:
 - events
 - monthly meetup
```

**Review by saucy-tech — CHANGES_REQUESTED** [review]

**Comment by saucy-tech** [comment]
@bitkarrot @bliotti @surfersol my thoughts on this is I'd prefer to keep meeting minutes in the github repo but include a repo link in our events blog post. We could add a sentence to the discord event monthly meetup saying "Checkout previous monthly meetup meetings here <url>.". This would then carryover when we get the python github actions setup (I'm working on it this weekend). 

We could spruce up the meeting minutes template if you wanted, and should reuse it month to month for consistency.

**Comment by bitkarrot** [comment]
> @bitkarrot @bliotti @surfersol my thoughts on this is I'd prefer to keep meeting minutes in the github repo but include a repo link in our events blog post. We could add a sentence to the discord event monthly meetup saying "Checkout previous monthly meetup meetings here .". This would then carryover when we get the python github actions setup (I'm working on it this weekend).
> 

That's a fantastic idea. I'm giving @saucy-tech idea a thumbs up. This way we have everything in one place. The event page is a good place to link to the git repo for meeting minutes. here's the location for the october meetup: https://github.com/plebnet-dev/website/blob/main/src/content/post/events.md#event-october-monthly-meetup

> We could spruce up the meeting minutes template if you wanted, and should reuse it month to month for consistency.

Sure!  Anybody want to create a template? @surfersol , @surfersol ? Definitely could use a meeting template, a good place to add it would be the meeting-notes repo. Also see the old meeting minutes as a point of reference: https://github.com/plebnet-dev/meeting-notes

File: src/content/post/surfersol_blogpost.md (hunk 2 above)
Comment (bitkarrot) [reply src/content/post/surfersol_blogpost.md:9]: @surfersol

**Comment by bitkarrot** [comment]
@surfersol I'd recommend making the Monthly Plebnet member contributions (the PRs) a separate blog post. Just keep the meeting-notes in the meeting repository.

**Comment by surfersol** [comment]
Modifying / moving the meeting minutes to https://github.com/plebnet-dev/meeting-notes

**Pull request closed without merging**